vos/docker/2.7/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:2.7-alpine
+
+ARG GIT_URL='https://github.com/opencadc/vostools.git'
+ARG GIT_BRANCH='master'
+
+RUN apk --no-cache add curl gcc git libffi-dev libxslt-dev make musl-dev openssl-dev tcsh
+
+RUN pip --disable-pip-version-check --no-cache-dir install pylint \
+    #
+    # Install pytest and mock
+    && pip --disable-pip-version-check --no-cache-dir install pytest mock \
```

Comment:
`pip install -r dev_requirements.txt` does that. It should be copied in the container and used to avoid duplicated effort.

Reply:
OK.